src/appkit.cs

```diff
@@ -18025,6 +18061,36 @@ interface NSTextFieldCell {
 		bool WantsNotificationForMarkedText { get; set; }
 	}
 
+	[BaseType (typeof(NSTextFieldCell))]
+	interface NSTokenFieldCell {
+		[Export ("initTextCell:")]
+		IntPtr Constructor (string aString);
+
```

Comment:
> When subclassing NSTextFieldCell you must implement all of the designated initializers. Those methods are: initWithCoder:, initTextCell:, and initImageCell:.

https://developer.apple.com/documentation/appkit/nstextfieldcell?language=objc

`initTextCell` is above, fine :)
`initWithCoder` comes from conformance with `NSCoding`, the generator handles that;
`initImageCell:` is likely from the base type and need to be re-exposed (since .net .ctor are not methods, like ObjC)

Reply:
I saw that. The header for NSTextFieldCell has 
```
- (instancetype)initImageCell:(nullable NSImage *)image NS_UNAVAILABLE; // Use the designated initializer initTextCell:
```

Reply:
Agreed, seems that `initImageCell:` is not needed, this can be manually verified trying to subclass in Xcode and add said ctor and try to call said super, it won't build.